src/components/GameInfoV2/GameInfoV2.module.scss

```diff
@@ -0,0 +1,245 @@
+@use '../../styles/utilities/loading' as *;
+
+.container {
+  display: flex;
+  gap: var(--space-xl);
+  justify-content: center;
+  align-items: space-between;
+  align-self: stretch;
+  padding: var(--space-2xl) var(--space-3xl);
+  background-color: var(--background-base);
```

Comment:
Isn't the background transparent in the designs?

Reply:
I saw that it was changed too because before it was filled